LEDs.cpp

```diff
@@ -0,0 +1,49 @@
+#include "LEDs.h"
+
+const uint8_t MAX_PICO_TX = 19;

```

Comment:
Is the Max or the Pico 19 for TX? We need a naming convention that we can easily tell the source of the signal.
Ex: PICO_MOSI_MAX7219